node/__tests__/unit.spec.ts

```diff
@@ -223,6 +274,7 @@ import { Radio } from "antd/es/radio/index.js";
 import { Input } from "antd/es/input/index.js";
 import React from "react";
 import ReactDOM from "react-dom";
+import { InputProps } from "antd";
```

Comment:
This seems not as expected since the code generation result doesn't contain the type reference to that or it's a big with SWC

Reply:
It seems a bug of SWC🧐. I didn't remove any type. But  `Props` in `class Page extends React.Component<Props,any>` disappear.

I add a new line `type Props = InputProps;` into test case. This line remains but only the class super type disappear😂

Reply:
LGTM